python_modules/dagster-graphql/dagster_graphql/schema/external.py

```diff
@@ -49,6 +49,14 @@
 GrapheneLocationStateChangeEventType = graphene.Enum.from_enum(LocationStateChangeEventType)
 
 
+class GrapheneDagsterLibrary(graphene.ObjectType):
```

Comment:
as in previous PR could stick some Metadata/Info/Version noun here potentially to clarify what this is